emission/core/get_database.py

```diff
@@ -320,4 +320,4 @@ def save(db, entry):
         # logging.debug("entry has id, calling with match, result = %s" % result.raw_result)
     else:
         result = db.insert_one(entry)
-        # logging.debug("entry has id, calling without match, result = %s" % result.inserted_id)
+        # logging.debug("entry has id, calling without match, result = %s" % result.inserted_id)
```

Comment:
extraneous whitespace?

Reply:
Yes, removed an extra empty line at the end.